petab/visualize/templates/mystyle.css

```diff
@@ -0,0 +1,25 @@
+/* Vertical table column headers */
+.vrt-header thead th {
+  writing-mode: vertical-lr;
+  min-width: 50px; /* for firefox */
+}
+
+.vrt-header tbody th {
+  white-space: nowrap;
+}
+
+table {
+  border-collapse: collapse;
+  text-align: right;
+}
+
+table, th, td {
+  border: 1px solid black;
+}
+
+ th {
+  background-color: #4CAF50;
+  color: white;
```

Comment:
Fine, if it's readable... (white on green?)

Reply:
For me its readable, although the color scheme is debatable... If somebody has his artistic moment... :)
Want to see how much it is used before investing much more time in styling.